docs/docs/ref-01-top-level-api.md

```diff
@@ -139,7 +139,7 @@ The `react-dom` package provides DOM-specific methods that can be used at the to
 ### ReactDOM.render
 
 ```javascript
-render(
+void render(
```

Comment:
If we omit return type then it still will be a source of confusion and inconsistency